scripts/remove-tags

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```

Comment:
Is there a reason we aren't using salt? It would minimize the amount of various scripts that need to be run (including as part of `sd-clean-all.sls`). We use salt for setting some workstation tags, for example: https://github.com/freedomofpress/securedrop-workstation/blob/5482f2e9c0e0658e0d44123e938bba49d40a0c04/dom0/sd-gpg.sls#L26-L28